test/cases/040_packages/032_bcc/check.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+nsenter -t 1 -m -i -u -n -p --  /bin/ash -c "/usr/share/bcc/tools/softirqs 1 1"
+if [ "$?" -eq "0" ]; then
+	printf "bcc test suite PASSED\n" > /dev/console
+	/sbin/poweroff -f
```

Comment:
If you run this as a `onboot` you can add the `linuxkit/poweroff` package to handle poweroff.